buildSrc/src/main/groovy/com/uber/okbuck/composer/android/LintRuleComposer.groovy

```diff
@@ -44,7 +41,17 @@ final class LintRuleComposer extends JvmBuckRuleComposer {
         if (customLintRules) {
             lintCmds.add("export ANDROID_LINT_JARS=\"${toLocation(customLintRules)}\";")
         }
-        lintCmds += ["mkdir -p \$OUT;", "RUN_IN=`dirname ${toLocation(fileRule(target.manifest))}`;", "exec java", "-Djava.awt.headless=true"]
+        lintCmds += ["mkdir -p \$OUT;", "RUN_IN=`dirname ${toLocation(fileRule(target.manifest))}`;"]
+
+        // Workaround till https://issuetracker.google.com/issues/64683008 is addressed
+        if (!target.main.sources.empty) {
+            lintCmds += [
+                    "CP_FILE=`sed \"s/@//\" <<< ${toClasspathFile(":${src(target)}")}`;",
+                    "sed -i.bak -e \"s|${target.rootProject.projectDir.absolutePath}/||g\" -e \"s|\\'||g\" \$CP_FILE;"
```

Comment:
ohhhh, idk idk, sed is pretty dangerous dependency to use because it works differently on GNU (Linux) and BSD (macOS) https://unix.stackexchange.com/a/131940